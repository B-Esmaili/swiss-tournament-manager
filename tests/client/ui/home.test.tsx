import { render } from '../../test-utils'
import Home from 'pages/index'

describe('Home page', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<Home />, {})
        expect(asFragment()).toMatchSnapshot()
    });
})