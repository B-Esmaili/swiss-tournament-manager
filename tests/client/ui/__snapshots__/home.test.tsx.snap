// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      Hello
    </div>
  </div>
</DocumentFragment>
`;
